environment.yml

```diff
@@ -115,6 +115,7 @@ dependencies:
   - pygments # Code highlighting
 
   - pip:
+      - dataframe-api-compat>=0.1.7
```

Comment:
Side question, will this be on conda-forge at some point?

Reply:
yup! once I figure that part out :)